contents/customers/rayfit.md

```diff
@@ -0,0 +1,66 @@
+date: 2025-11-21
+---
+
+[Rayfit](https://www.rayfit.com/) is building an AI personal training app that solves for the hardest part of fitness: showing up consistently. Their conversational AI coach, _Ray_, generates personalized workouts based on equipment, energy level, and available time.
+
+Alan, Rayfit’s Full Stack Engineering Lead, manages product development and data infrastructure. We asked him why he chose PostHog:
+
+"As an early stage startup, we're cost sensitive," Alan says. “Initially we planned to stitch together an analytics tool and Metabase, but then we looked at PostHog. It's just so cheap and easy. We connected our database and built our pipeline with almost no work."
+
+For a team deep in the technical trenches of machine learning and computer vision, PostHog's "one-stop shop" setup meaningfully reduced the time and effort of building out their stack. [Analytics](/docs/product-analytics), [feature flags](/docs/feature-flags), and a [data warehouse](/docs/data-warehouse) could all live one in place.
+
+That solved the infrastructure problem. Next was ensuring the entire team was on board.
+
+## Making data accessible with PostHog AI
+
+For Rayfit, the early friction wasn’t collecting data, it was making sure everyone could access it. 
+
+"PostHog can present as really technical," Alan explains. “Our CEO and designer had used Mixpanel and Amplitude before — which felt more approachable since PostHog is built for engineers.”
+
+Like many, their in-app data was also tricky to access. It requires [SQL skills](/product-engineers/sql-for-analytics) not everyone has. "A lot of the data we use for the generated workouts lives in the app itself," Alan says. "It wouldn't make sense to send a unique event for each one. With the warehouse, we can join it to a person and learn more about their behaviors."
+
+That’s where [PostHog AI](/ai) comes in. For Alan, SQL is second nature — AI just removes some friction. For the rest of the team, having AI draft and edit queries makes the whole workflow far more accessible. “PostHog AI has actually been a deciding factor in our decision to stick with PostHog,” he told us. “Without it, we probably would have moved to a different analytics package because the learning curve felt so steep.”
+
+<OSQuote
+  customer="rayfit"
+  author="alan_yang"
+  quote={0}
+/>
+
+## Realistic expectations, real results
+
+As an early adopter of AI products (in and outside of PostHog), Alan is pragmatic about AI capabilities. “All LLMs right now are about 30–50%,” he says. “But it’s a nice problem space because that’s still a win when it gets it right.”
+
+So when PostHog AI doesn’t nail the syntax on the first try, he doesn’t see it as a failure — just part of the workflow. In fact, it mirrors how both Rayfit and PostHog operate. “We're shipping things before they're fully baked just like PostHog,” he says. “So when AI features don't work perfectly, everyone is willing to try again.”
```

Comment:
> We're shipping things before they're fully baked just like PostHog

Could we reframe this as "we're shipping to learn" rather than "fully baked" which can come off a reckless to some readers?